### DISCUSSION TITLE
Patch Flux extension manifests/controllers

### DISCUSSION THREAD

**Discussion by pinkfloydx33** [Azure/AKS#3229] [Q&A]
Is it possible to patch Flux-system controllers/customize manifests when using the Microsoft.flux (gitops) AKS extension?

For some workloads, such as Image Automation with reflector, it may be required to patch A. the controller arguments to enable auto Acr login and B. the pod labels for aad-pod-identity

Image reflector is disabled by default but can be enabled using  the cli via configuration settings for the extension. However there doesn't seem to be a way to patch these system controllers similar to the way you would if you bootstrapped Flux directly https://fluxcd.io/flux/guides/image-update/#using-native-azure-acr-auto-login

**Comment by joshuadmatthews** [comment]
@pinkfloydx33  Did you ever figure this out? Running up against the same thing currently.

**Reply by pinkfloydx33** [reply]
No I did not
